neard/src/runtime.rs

```diff
@@ -216,7 +222,8 @@ impl NightshadeRuntime {
         epoch_manager.get_epoch_info(&epoch_id).map(|info| info.epoch_height).map_err(Error::from)
     }
 
-    fn genesis_state_from_dump(store: Arc<Store>, home_dir: &Path) -> (Arc<Store>, Vec<StateRoot>) {
+    fn genesis_state_from_dump(store: Arc<Store>, home_dir: &Path) -> Vec<StateRoot> {
+        error!(target: "near", "Loading genesis from a state dump file. Do not use this outside of genesis-tools");
```

Comment:
Target "near" looks odd here; is it on puprose? We use "runtime" here.

Why is it an error?

Reply:
This code is bad and should not be used by anyone. Seems like there is something in genesis-tools that still uses it, but otherwise we should remove it.

The reasoning was that this is genesis computation that happens once at the first start of the node, but can use runtime

Reply:
Actually for some reason target runtime does not show up for me, maybe it's disabled by default

Reply:
It could be the case, and that is fine. If you want to make something visible to the users of neard, I suggest you display the message from neard, not the runtime. The usage warning should go into the docstrings or deprecation attribute.

Reply:
this is neard crate